src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,33 @@
+import s from './HomePage.module.scss';
+import WorldMap from '../../../public/img/World Map.svg';
+import Button from '../Button/Button';
+
+export const HomePage = () => {
+  return (
+    <div className={s.container}>
+      <div className={s.titleContainer}>
+        <h5 className={s.title}>
+          Empower with <br />
+          <h5 className={s.subtitle}>AQE Educate.</h5>
+        </h5>
+
+        <p className={s.paragraph}>
+          Your trusted partner in professional development <br /> for educators.
```

Comment:
This < br /> might be causing a horizontal scroll since text is too wide
<img width="596" alt="Zrzut ekranu 2024-07-25 o 15 28 31" src="https://github.com/user-attachments/assets/77e01b55-31d0-480d-83f7-7b0e0bebb0a4">